packages/ui/src/components/primitives/Input.tsx

```diff
@@ -53,66 +54,64 @@ type StyledInputCssVariants = NonNullable<
   Parameters<(typeof StyledInputCss)['raw']>['0']
 >
 
-const Input: FC<
+const Input = forwardRef<
+  HTMLInputElement,
   Omit<HTMLProps<HTMLInputElement>, 'size'> &
     PropsWithChildren & {
       icon?: ReactNode
       iconPosition?: 'left' | 'right'
       iconCss?: Styles
       containerCss?: Styles
-      inputRef?: (element: HTMLInputElement) => void
```

Comment:
I see we removed inputRef, was it not used anywhere else?

Reply:
Updated it to pass the ref directly into the component like `<Input ref={ref} `